Разметка всех страниц

Bravenko opened the pull request:
---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss-individual/1/user/2512403/tasks/4)

:boom: https://htmlacademy-htmlcss.github.io/2512403-sedona-1/2/

Bravenko committed:
Создает хедер

Создает хедер и мейн

Bravenko committed:
Делает разметку проекта

Bravenko committed:
делает разметку проекта

Bravenko committed:
Добавляет изображение

Добавил изображение гор на главную страницу

Bravenko committed:
Заменяет ссылки на кнопки

В карточках товаров заменил ссылки "избранное" на кнопки

Bravenko committed:
Корректирует недочеты

Откорректировал недочеты, возникшие при проверке основной страницы в валидаторе

Bravenko committed:
Корректирует недочеты

Откорректировал недочеты, возникшие при проверке кода каталога в валидаторе

Bravenko committed:
Корректирует хедер

В хедере главной страницы обернул логотип в ссылку

Bravenko committed:
Заменяет вид списка

Заменил список пагинации с неупорядоченного на упорядоченный

Bravenko committed:
Убирает пункт из списка

Убрал из списка пункт 10 и сделал его отдельной ссылкой

Bravenko committed:
Добавляет параграфф

Добавил пропущенный абзац с результатами поиска

vadimrahmanov commented on index.html:
```diff
@@ -5,8 +5,150 @@
   </head>
   <body>
+    <header class="page-header">
+      <a class=logo-link href="">
```
Здесь можно полностью убрать атрибут "href" т.к мы уже находимся на главной странице

vadimrahmanov commented on index.html:
```diff
@@ -5,8 +5,150 @@
+      <nav class="navigation">
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <a class="navigation-link navigation-link-current" href="index.html">Главная</a>
```
Здесь тоже можно полностью убрать атрибут href

vadimrahmanov commented on index.html:
```diff
@@ -5,8 +5,150 @@
+            <a class="navigation-link navigation-link-current" href="index.html">Главная</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="about.html">О нас</a>
```
у тебя не будет этой страницы, поэтому можно просто # поставить в атрибут

vadimrahmanov commented on index.html:
```diff
@@ -5,8 +5,150 @@
+      <img class="greetings-picture" src="#" alt="welcome">
+      <img class="greetings-picture" src="#" alt="to the gorgeous">
+      <img class="greetings-picture" src="#" alt="sedona, because the grand canyon is sucks!">
+      <img class="greetings-picture" src="#" alt="горы">
```
Это всё одна картинка кроме гор, горы это фоновое изображение, которое не должно быть в разметке.

vadimrahmanov commented on index.html:
```diff
@@ -5,8 +5,150 @@
+        <h2 class="greetings-description">Седона — небольшой городок в Аризоне,<br>
+          заслуживающий большего!</h2>
+        <p class="greetings-description">Рассмотрим причины, по которым Седона круче, чем Гранд-Каньон!</p>
+      </section >
```
у трёх разных элементов одинаковый класс. Они даже в макете выглядят по-разному.

vadimrahmanov commented on index.html:
```diff
@@ -5,8 +5,150 @@
+        <h2 class="visually-hiden">Преимущества</h2>
+        <ul class="advantages-list">
+          <li class="advantages-item">
+            <h3 class="advantages-title">НАСТОЯЩИЙ ГОРОДОК</h3>
```
Это замечание касается всех мест, где ты используешь все буквы заглавные. Не нужно писать капсом. Этот эффект потом делается через CSS.

vadimrahmanov commented on index.html:
```diff
@@ -5,8 +5,150 @@
+            <img src="#" alt="Вид на каньон и мост дьявола">
+            <p class="advantages-description">Да, по нему можно пройти! Если вы осмелитесь, разумеется</p>
+          </li>
+        </ul>
```
здесь конец секции, а ниже уже другая секция.

vadimrahmanov commented on index.html:
```diff
@@ -5,8 +5,150 @@
+          </li>
+        </ul>
+      </section>
+      <section class="Hotel-search">
```
Не нужно использовать заглавные буквы в названии классов

vadimrahmanov commented on index.html:
```diff
@@ -5,8 +5,150 @@
+            <span class="visually-hidden">Подписаться</span>
+          </button>
+        </form>
+      </article>
```
Это просто секция, поэтому здесь уместнее будет использовать тэг section

vadimrahmanov commented on index.html:
```diff
@@ -5,8 +5,150 @@
+      <article class="subscribe">
+        <h2 class="subscribe-invitation">Подпишитесь на рассылку</h2>
+        <p>Только полезная информация и никакого спама, честное бойскаутское!
+        </p>
```
нету необходимости переносить закрывающий тэг на новую строку

vadimrahmanov commented on index.html:
```diff
@@ -5,8 +5,150 @@
+          <label class="visually-hiden" for="newsletter-email">Email</label>
+          <input class="field"  placeholder="ваш e-mail" type="email" name="newsletter-email" id="newsletter-email" required>
+          <button class="button newsletter-button" type="submit">
+            <span class="visually-hidden">Подписаться</span>
```
Эта надпись не скрытая, поэтому не нужно её выносить отдельно в span с таким классом

vadimrahmanov commented on index.html:
```diff
@@ -5,8 +5,150 @@
+      </article>
+    </main>
+
+    <footer class="page-footer"><ul class="navigation-list">
```
ul нужно перенести на новую строку

vadimrahmanov commented on index.html:
```diff
@@ -5,8 +5,150 @@
+      <a class="contacts-phone" href="tel:8(812)812-12-12">8(812)812-12-12</a>
+    </address>
+    <img class="logo" src="#" alt="html academy">
+    <img class="logo" src="#" alt="html academy">
```
логотип один, здесь дублируется тэг

vadimrahmanov commented on catalog.html:
```diff
@@ -0,0 +1,244 @@
+          <a class="navigation-link" href="main.html">Главная</a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link" href="about.html">О нас</a>
```
поставь заглушку # вместо ссылки

vadimrahmanov commented on catalog.html:
```diff
@@ -0,0 +1,244 @@
+          <a class="navigation-link" href="about.html">О нас</a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link navigation-link-current" href="catalog.html">Гостиницы</a>
```
здесь можно убрать атрибут href

vadimrahmanov commented on catalog.html:
```diff
@@ -0,0 +1,244 @@
+            <a class="breadcrumbs-link">Каталог</a>
+          </li>
+        </ul>
+    </header>
```
У страницы не должно быть два header, исключение - статьи с тэгом article

vadimrahmanov commented on catalog.html:
```diff
@@ -0,0 +1,244 @@
+      <h1 class="inner-header-title">Гостиницы Седоны</h1>
+        <ul class="breadcrumbs">
+          <li class="breadcrumbs-item">
+            <a class="breadcrumbs-link" href="index.html"><img class="logo" src="#" alt="Главная">
```
перенеси изображение на новую строку

vadimrahmanov commented on catalog.html:
```diff
@@ -0,0 +1,244 @@
+            <ul class="catalog-filter-list">
+              <li class="catalog-filter-item">
+                <label class="control">
+                  <input class="control-input" type="checkbox" name="bed-head-for-men" checked>
```
lifehack: Можно сделать name="infrastructure" и value="всё_что_у_тебя_сейчас_в_name"

vadimrahmanov commented on catalog.html:
```diff
@@ -0,0 +1,244 @@
+         </a>
+         <a class="button" href="#">
+          <span class="visually-hidden">Показать карточки</span>
+         </a>
```
это всё кнопки (а не ссылки)

vadimrahmanov commented on catalog.html:
```diff
@@ -0,0 +1,244 @@
+           <button class="button-current" type="submit">Применить</button>
+          <button class="button" type="submit">Сбросить</button>
+          </form>
+        <section>
```
нету заголовка

vadimrahmanov commented on catalog.html:
```diff
@@ -0,0 +1,244 @@
+         <a class="button" href="#">
+          <span class="visually-hidden">Показать карточки</span>
+         </a>
+         <ul class="product-card">
```
у списка карточек и карточки должны быть разные классы

vadimrahmanov commented on catalog.html:
```diff
@@ -0,0 +1,244 @@
+           <b class="product-card-price">Гостиница</b>
+           <b class="product-card-price">От 4000 ₽</b>
+           <a class="product-card-button button" href="#">Подробнее</a>
+           <button class="button-favorites" type="button">В избранное</button>
```
Отступы

vadimrahmanov commented on catalog.html:
```diff
@@ -0,0 +1,244 @@
+            <h3 class="product-card-title">Amara Resort & Spa</h3>
+           </a>
+           <b class="product-card-price">Гостиница</b>
+           <b class="product-card-price">От 4000 ₽</b>
```
вместо b используй strong, т.к "b" устарел

vadimrahmanov commented on catalog.html:
```diff
@@ -0,0 +1,244 @@
+          <span class="visually-hidden">Подписаться</span>
+        </button>
+      </form>
+    </article>
```
такие же замечания как и на главной

vadimrahmanov commented on catalog.html:
```diff
@@ -0,0 +1,244 @@
+    <a class="contacts-phone" href="tel:8(812)812-12-12">8(812)812-12-12</a>
+  </address>
+  <img class="logo" src="#" alt="html academy">
+</footer>
```
такие же замечания как на главной

vadimrahmanov commented on catalog.html:
```diff
@@ -0,0 +1,244 @@
+            <a class="pagination-link" >...</a>
+          </li>
+        </ol>
+        <a class="pagination-link" href="#">10</a>
```
почему эта кнопка не в списке?

vadimrahmanov commented on catalog.html:
```diff
@@ -0,0 +1,244 @@
+            <a class="pagination-link" href="#">4</a>
+          </li>
+          <li class="pagination-item">
+            <a class="pagination-link" >...</a>
```
не совсем ссылка

Bravenko committed:
Убирает ссылку

Убрал ссылку на главную страницу в шапке главной страницы

Bravenko committed:
Убирает ссылку

Убрал ссылку на главную страницу в навигации шапки

Bravenko committed:
Заменяет ссылку

Заменил ссылку на страницу, которой не будет в проекте

Bravenko committed:
Редактирует изображения

Вместо трех изображений сделал одно, фоновое изображение

Bravenko committed:
Редактирует текст

Заменил весь текст, прописанный заглавными буквами на строчные

Bravenko committed:
Разбивает секцию

Вынес часть элементов, секции "Преимущества" в отдельную секцию

Bravenko committed:
Убирает заглавные буквы

Убрал заглавные буквы из названий классов

Bravenko committed:
Заменяет article

Заменил article на section

Bravenko committed:
Переносит закрывающий тег

Перенес закрывающий тег на начальную строку

Bravenko committed:
Меняет класс

Заменил класс у span

Bravenko committed:
Переносит ul

Перенес ul на новую строку

Bravenko committed:
Убирает изображение

Убрал задублированное изображение

Bravenko committed:
Корректирую ссылку

Вместо ссылки на страницу " О Седоне" ставлю заглушку

Bravenko committed:
Убирает ссылку

Убрал ссылку на текущую страницу

Bravenko committed:
Убирает header и переносит изображение

Убрал лишний heder, заменил его на section
Перенес изображение на другую строку

Bravenko committed:
Заменяет name

Заменил name и присвоил value

Bravenko committed:
Меняет ссылки на кнопки

Заменил ссылки на кнопки

Bravenko committed:
Корректирует секции

Разбил секцию на две разных, добавил имя

Bravenko committed:
Изменяет имена

Присвоил карточкам имена, отличные от списка

Bravenko committed:
Заменяет теги

Заменил b на strong

Bravenko committed:
Убирает article

Заменил article на section

Bravenko committed:
Переносит список

Переносит список на другую строку

Bravenko committed:
Добавляет ссылку в список

Bravenko committed:
Заменяет ссылки

Заменил в пагинации ссылки на кнопки

Bravenko committed:
Присваивает разные классы

Откорректировал названия классов, которые шли под одним именем

Bravenko committed:
Добавляет отступы

Bravenko committed:
Добавляет рейтинг

Добавил карточкам отелей количество звезд и рейтинг ( не уверен, что сделал это правильно)

vadimrahmanov commented on catalog.html:
```diff
@@ -0,0 +1,257 @@
+           </li>
+          </ul>
+        </section>
+     <section class="catalog-products">
```
это скорее sort and filter, нежели products

vadimrahmanov commented on catalog.html:
```diff
@@ -0,0 +1,257 @@
+
+  </main>
+
+
```
убери везде такие ненужны переводы строки

vadimrahmanov commented on catalog.html:
```diff
@@ -0,0 +1,257 @@
+            <button class="pagination-button pagination-current">10</button>
+          </li>
+        </ol>
+    </section>
```
проблемы с тэгами. Есть закрывающий, а открывающего нету. Также проблема с отступами
